release-content/0.14/release-notes/13317_12502_Remove_limit_on_RenderLayers.md

```diff
@@ -1,4 +1,12 @@
 <!-- #12502 Remove limit on RenderLayers. -->
 <!-- https://github.com/bevyengine/bevy/pull/13317 -->
 
-<!-- TODO -->
+In Bevy you are able to assign entities spawned to different layers.
+
+You can make 3D models, meshes, lights, etc. all have their own if you wish.
+
+You can tell cameras which layers to render, and only objects which are part of those layers get rendered to that camera's target.
```

Comment:
```suggestion
Render layers are used to quickly toggle the visibility of sets of objects, and control which objects can be seen by which cameras.
This can be useful for things like debug views, gear preview screens, toggleable diegetic UI and so on.
```